brave_domains/BUILD.gn

```diff
@@ -6,15 +6,40 @@
 import("//build/buildflag_header.gni")
 import("buildflags.gni")
 
-# TODO(petemill): Default target for this project could instead be some cpp
-# helpers which create domain names for consumers for the correct environment
-# and can vary based on dynamic cli param overrides.
+source_set("brave_domains") {
```

Comment:
The placement was @bridiver 's suggestion - we can move it around if we all agree

Reply:
@goodov this is based on google having essentially the same thing at the top level

Reply:
> @goodov this is based on google having essentially the same thing at the top level

you mean `google_apis`? if so, then they declare it as `component()`, so logically they should also put it in `components` instead of the root directory.